pipelines/skylab/build_indices/BuildIndices.wdl

```diff
@@ -42,6 +42,7 @@ task GetReferences {
   runtime {
     docker: "quay.io/humancellatlas/secondary-analysis-star:v0.2.2-2.5.3a-1.0.0"
     disks: "local-disk 10 HDD"
+        memory: "50 GiB"
```

Comment:
That's a lot of memory - are you sure that's all needed for wget and gunzip?
Also - the spacing is off.  memory should align with disks.